tests/conftest.py

```diff
@@ -30,6 +30,11 @@ async def empty_blockchain(request):
     db_path.unlink()
 
 
+@pytest.fixture(scope="function", params=[1, 2])
+async def db_version(request):
+    yield request.param
```

Comment:
Kinda doesn't matter, but...

```suggestion
def db_version(request):
    return request.param
```